media-libs/tiff/files/tiff-4.5.0-CVE-2022-48281.patch

```diff
@@ -0,0 +1,14 @@
+Index: tiff-4.5.0/tools/tiffcrop.c
```

Comment:
Two notes:
1. I've run the `scrub-patch` tool from iwdevtools on this when committing
2. I've added a link to the relevant Gentoo bug and commit upstream at the top of the file too (see https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto)

(Know the refs are in commit message, but having it in the patch (as well) makes it quick to lookup.)

Reply:
Thanks! I'll keep it in mind for next time.